config/routes.rb

```diff
@@ -1,26 +1,17 @@
 Rails.application.routes.draw do
+  
   scope "(:locale)", locale: /en|vi/ do
     # Trang chủ
     root to: "static_pages#home"
+    devise_for :users,
```

Comment:
```suggestion
    devise_for :users, only: [:sessions, :registrations, :passwords, :confirmations]
```
E giới hạn các router lại nha